arelle/plugin/validate/ESEF/ESEF_Current/Image.py

```diff
@@ -94,9 +94,10 @@ def validateImage(
                     _("Images SHOULD be included in the XHTML document as a base64 encoded string unless their size exceeds the minimum size for the authority (%(maxImageSize)s): %(file)s."),
                     modelObject=elt, maxImageSize=minExternalRessourceSize, file=os.path.basename(normalizedUri), evaluatedMsg=evaluatedMsg)
         except IOError as err:
+            fileReferencingImage = os.path.basename(baseUrl) if baseUrl else ''
             modelXbrl.error(f"{contentOtherThanXHTMLGuidance}.imageFileCannotBeLoaded",
-                            _("Error opening the file '%(src)s': %(error)s"),
-                            modelObject=elt, src=image, error=err, evaluatedMsg=evaluatedMsg)
+                            _("Error opening the file '%(src)s' referenced by '%(fileReferencingImage)s': %(error)s"),
+                            modelObject=elt, src=image, fileReferencingImage=fileReferencingImage, error=err, evaluatedMsg=evaluatedMsg)
```

Comment:
The `modelObject=elt` is confusing for SVGs because it reports the error against element IDs _inside_ the SVG XML rather than the `<img>` from the IXDS that referenced the SVG.  It seems like `checkSVGContentElt` needs to propagate its outer `elt` down into `validateImage` for use as `modelObject` in the error, and then add some different attribute for this `elt`.

Reply:
I updated the code to include refs for all linking images with working hrefs for SVGs.

For example:
`test-2024-01-12-en.xhtml` > `main.svg` > `image-reference.svg` > `../../missing.jpg` (missing) 

produces:
```xml
<entry code="ESEF.2.5.1.imageFileCannotBeLoaded" level="error">
 <message src="../../missing.jpg" fileReferencingImage="image-reference.svg" error="[Errno 2] Archive /Users/austinmatherne/Downloads/test-2024-01-12-en.zip/test-2024-01-12-en/reports/_IXDS#?#/Users/austinmathern..." evaluatedMsg="">[ESEF.2.5.1.imageFileCannotBeLoaded] Error opening the file '../../missing.jpg' referenced by 'image-reference.svg': [Errno 2] Archive /Users/austinmatherne/Downloads/test-2024-01-12-en.zip/test-2024-01-12-en/reports/_IXDS#?#/Users/austinmatherne/Downloads/test-2024-01-12-en.zip/test-2024-01-12-en/reports/test-2024-01-12-en.xhtml: 'missing.jpg' - image-reference.svg 2, main.svg 2, test-2024-01-12-en.xhtml 512</message>
 <ref href="image-reference.svg#element(/1/1)" sourceLine="2">
  <property name="QName" value="image"/>
  <property name="href" value="../../missing.jpg"/>
  <property name="x" value="0"/>
  <property name="y" value="0"/>
  <property name="width" value="200px"/>
  <property name="height" value="200px"/>
 </ref>
 
 <ref href="main.svg#element(/1/1)" sourceLine="2">
  <property name="QName" value="image"/>
  <property name="href" value="image-reference.svg"/>
  <property name="x" value="0"/>
  <property name="y" value="0"/>
  <property name="width" value="200px"/>
  <property name="height" value="200px"/>
 </ref>
 
 <ref href="test-2024-01-12-en.xhtml#element(/1/2/2/1/10/1/1/1)" sourceLine="512">
  <property name="QName" value="img"/>
  <property name="src" value="main.svg"/>
  <property name="alt" value="main.svg"/>
  <property name="style" value="height: 300pt; width: 300pt"/>
 </ref></entry>
```